tests/components/roku/test_remote.py

```diff
@@ -0,0 +1,70 @@
+"""The tests for the Roku remote platform."""
+from homeassistant.components.remote import (
+    ATTR_COMMAND,
+    DOMAIN as REMOTE_DOMAIN,
+    SERVICE_SEND_COMMAND,
+)
+from homeassistant.const import ATTR_ENTITY_ID, SERVICE_TURN_OFF, SERVICE_TURN_ON
+from homeassistant.helpers.typing import HomeAssistantType
+
+from tests.async_mock import patch
+from tests.components.roku import UPNP_SERIAL, setup_integration
+from tests.test_util.aiohttp import AiohttpClientMocker
+
+MAIN_ENTITY_ID = f"{REMOTE_DOMAIN}.my_roku_3"
+
+# pylint: disable=redefined-outer-name
+
+
+async def test_setup(
+    hass: HomeAssistantType, aioclient_mock: AiohttpClientMocker
+) -> None:
+    """Test setup with basic config."""
+    await setup_integration(hass, aioclient_mock)
+    assert hass.states.get(MAIN_ENTITY_ID)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)